cvat/apps/engine/backup.py

```diff
@@ -504,8 +505,9 @@ def _write_data(zip_object):
         else:
             uploaded_files = _write_data(self._file)
 
-        data['use_zip_chunks'] = data.pop('chunk_type') == DataChoice.IMAGESET
+        use_zip_chunks = data.pop('use_zip_chunks', False) or data.pop('chunk_type') == DataChoice.IMAGESET
         data = data_serializer.data
+        data['use_zip_chunks'] = use_zip_chunks
```

Comment:
Sorry for my mistake. But I've taken a look at this one more time and these lines will be enough to fix. 
```suggestion
        use_zip_chunks = data.pop('chunk_type') == DataChoice.IMAGESET
        data = data_serializer.data
        data['use_zip_chunks'] = use_zip_chunks
```